.manylinux-install.sh

```diff
@@ -7,7 +7,8 @@ for PYBIN in /opt/python/*/bin; do
     if [[ "${PYBIN}" == *"cp27"* ]] || \
        [[ "${PYBIN}" == *"cp35"* ]] || \
        [[ "${PYBIN}" == *"cp36"* ]] || \
-       [[ "${PYBIN}" == *"cp37"* ]]; then
+       [[ "${PYBIN}" == *"cp37"* ]] || \
+       [[ "${PYBIN}" == *"cp38"* ]]; then
```

Comment:
And now we will learn if the manylinux docker images have already been updated to contain Python 3.8!  :)

(I've no idea who maintains those images.)

Reply:
Ah, we won't because manylinux builds are for git tags only.